crates/viewer/re_viewer/src/app_blueprint.rs

```diff
@@ -219,10 +220,12 @@ impl AppBlueprint<'_> {
 
             let timepoint = store_ctx.blueprint_timepoint_for_writes();
 
+            // Can only fail if we have the wrong number of instances for the component, and we don't
```

Comment:
~~need to clean this comment as well~~

---

I would like to remove this comment & `#[allow(clippy::unwrap_used)]` in favor of using `.expect()` here as well.